lit/docs/pipelines/instance-groups.lit

```diff
@@ -0,0 +1,212 @@
+\title{Grouping Pipelines}{instance-groups}
```

Comment:
While we work out a good name for this, can we have a stable tag like `pipeline-instances` or `instanced-pipelines`? That way we can at least start linking to it without breaking the URL later.